gremlin-go/build/generate.groovy

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
I noticed that when i build locally, this script modifies the `gremlin.go` file - the formatting is apparently different from what is being committed. I'm wondering if someone's IDE perhaps formatted the file before commit? I assume we want the format committed that is produced by this file so that it doesn't create unnecessary commits and file contention?

Reply:
Yeah I think IntelliJ auto formats it if you commit through IntelliJ. I will fix this.